site/src/pages/SetupPage/SetupPage.test.tsx

```diff
@@ -52,30 +53,33 @@ describe("Setup Page", () => {
     let userHasBeenCreated = false;
 
     server.use(
-      rest.get("/api/v2/users/me", (req, res, ctx) => {
-        if (!userHasBeenCreated) {
-          return res(ctx.status(401), ctx.json({ message: "no user here" }));
-        }
-        return res(ctx.status(200), ctx.json(MockUser));
-      }),
-      rest.get("/api/v2/users/first", (req, res, ctx) => {
-        if (!userHasBeenCreated) {
-          return res(
-            ctx.status(404),
-            ctx.json({ message: "no first user has been created" }),
-          );
-        }
-        return res(
-          ctx.status(200),
-          ctx.json({ message: "hooray, someone exists!" }),
-        );
-      }),
-      rest.post("/api/v2/users/first", (req, res, ctx) => {
+      http.get(
+        "/api/v2/users/me",
+        async (): Promise<StrictResponse<User | Response>> => {
```

Comment:
Was reading up on how the `StrictResponse` type works, and saw this thread (not making this a link to avoid pinging them):
```txt
https://github.com/mswjs/msw/issues/1792
```
Namely this section:
> I recommend using the generics for this. That way you get a complete type-safe request handler. By annotating the return type of the resolver alone, you are tapping into implementation details (StrictResponse), which is not a future-proof strategy.

Looks like `StrictResponse` might not be something they intend for app devs to use

Reply:
Something like this wouldn't work?
```tsx
http.get<never, RequestBody, User | Response>(/* Unchanged code */);
```

Reply:
Ahh I see, it worked nicely with `http.get<never, null, User | Response>`